stable/scalyr/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for scalyr.
+image:
+  repository: scalyr/scalyr-docker-agent
+  tag: 2.0.14
+  pullPolicy: IfNotPresent
+
+scalyr:
+  ## You'll need to set this to your Scalyr API key before the agent will run.
+  ## ref: https://www.scalyr.com
+  ##
+  # apiKey:
+
+resources:
```

Comment:
Make resources configurable but do not specify defaults. This should be a conscious choice of the user.